app/questionnaire/router.py

```diff
@@ -144,7 +145,9 @@ def get_next_location_url(
         # Due to backwards routing you can be on the last block of the path but with an in_progress section
         is_last_block_on_path = routing_path[-1] == location.block_id
         if is_last_block_on_path:
-            return self._get_first_incomplete_location_in_section(routing_path).url()
+            # The section is not complete therefore we must have a location
```

Comment:
Maybe we should prefix comment for type ignore.

i.e
```py
# Type ignore: The section is not complete therefore we must have a location
```

Reply:
Sounds like a good idea, eaiser to understand what we are commenting. 👍  Added